packages/cozy-mespapiers-lib/src/components/ModelSteps/Contact.jsx

```diff
@@ -51,7 +50,7 @@ const Contact = ({
         <Avatar size="small" style={styleAvatar} />
       </ListItemIcon>
       <ListItemText
-        primary={`${getFullname(contact)} ${
+        primary={`${getContactName(contact)} ${
```

Comment:
Je suggère plutôt de faire ceci :
`const { getDisplayName, getDefaultSortIndexValue } = models.contact`
```suggestion
        primary={`${getDisplayName(contact) || getDefaultSortIndexValue(contact)?.split('@')[0]} ${
```

Reply:
Chez moi, `getDisplayName` affiche l'adresse mail pour l'utilisateur courant et non pas son nom, c'est pour ça que j'ai appelé `getFullname` en premier dans `getContactName`
Par conséquent avec ta suggestion, `getDefaultSortIndexValue` n'est pas appelé pour l'utilisateur courant
De manière générale, `getDefaultSortIndexValue` couvre moins de cas que `getDisplayName` donc ça n'apporterait rien en plus
Et ce n'a pas vraiment l'air d'être fait pour être exposé à l'utilisateur : pour un contact qui a juste un prénom "Prénom" et un nom "NOM", ça va afficher "nomprénom" alors que `getFullname` et `getDisplayName` vont afficher "Prénom NOM", ce qui me semble plus adapté

Reply:
- Pour `getDisplayName`, il retourne l'email si il n'y a pas de nom/prénom renseignés => Ok (on pas les inventés^^)
- Pour `getDefaultSortIndexValue`, après recherche, je dirais qu'il est même inutile, car `getDisplayName` passe par là également

(En environnement de dev, il ne sait pas retourner le pseudo du Cozy)